interpreter/interpreter_test.go

```diff
@@ -1387,7 +1406,17 @@ func TestInterpreter(t *testing.T) {
 				"track":      TrackState(state),
 			}
 			for mode, opt := range opts {
-				prg, vars, err = program(t, &tc, opt)
+				opts := []InterpretableDecorator{opt}
+				if tc.extraOpts != nil {
```

Comment:
Can you add the same block of code to the Benchmark as well?

Reply:
Before:

```
BenchmarkInterpreter/matches
BenchmarkInterpreter/matches-8                                                      	  165296	      7135 ns/op	    5845 B/op	
```

After:
```
BenchmarkInterpreter/matches
BenchmarkInterpreter/matches-8                                                      	  898917	      1413 ns/op	     210 B/op	
```